openshift/template.yaml

```diff
@@ -101,6 +101,24 @@ objects:
                     name: assisted-installer-rds
               - name: IMAGE_BUILDER_CMD
                 value: ""
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: bm-inventory
```

Comment:
it should be assisted service and not bm-inventory

Reply:
The pod's name is `bm-inventory` so I just used the same name, in addition this is taken from `bm-inventory:deploy/bm-inventory-service.yaml`

1. do you still want me to change it to `assisted-service`?
2. do we need to change the pod's name as well to `assisted-service`?
3. do we need to change the service name in `bm-inventory` as well?

Reply:
it should be assisted service in all places and not just the service name. we don't want to stuck with the bm-inventory for ever